src/components/BotAppearance/ColorPicker.tsx

```diff
@@ -0,0 +1,73 @@
+import {
```

Comment:
Can we convert the component to be a shared `components/common/FormColorPicker`?